### DISCUSSION TITLE
unable to use png in kitty with neofetch.

### DISCUSSION THREAD

**Discussion by nipundrall** [kovidgoyal/kitty#7370] [Q&A]
hello people,  i want to add a png in neofetch so i used the kitty icat.  When i use this command  ""  setsid kitten icat --stdin=no --use-window-size $COLUMNS,$LINES,3000,2000 --transfer-mode=file /home/nipun/.config/neofetch/pngs/doremon.png "" in terminal i get i image but cover the whole screen
So i changed it to "" setsid kitten icat --stdin=no --use-window-size $COLUMNS,$LINES,151,146 --transfer-mode=file /home/nipun/.config/neofetch/pngs/doremon.png ""  i get a smaller image the size i want. 
But when i paste this in neofetch config i get error "" Invalid size specification: ,,3000,2000 with error: strconv.Atoi: parsing "": invalid syntax "" but when i remove the window size flag i get a image in neofetch but it covers the whole screen.  and i know that those are pixels size but i can't get it to work with neofetch. please give me hint where i'm wrong.

**Comment by nipundrall** [comment]
Oh Nevermind it somehow works again

**Reply by KukicVidan** [reply]
fix?
